src/components/Common/Header.js

```diff
@@ -133,7 +140,14 @@ function Header() {
                 className={styles.tokenIcons}
                 id={open ? styles.show : styles.hide}
               >
-                <BiBell size="2vw" className={styles.bell} />
+                <div className={styles.alam}>
+                  <BiBell
+                    size="2vw"
+                    onClick={alamOpen}
+                    className={styles.bell}
```

Comment:
```suggestion
                    onClick={alamOpen}
                    className={styles.bell}
```
styles.bell을 클래스명으로 사용중이니 scss 파일에서 size를 적용해주면 될 것 같아요